pkgs/applications/editors/gnome-inform7/default.nix

```diff
@@ -0,0 +1,94 @@
+{ lib, stdenv, fetchFromGitHub, meson, ninja, pkg-config, python3, perl, bison
+, texinfo, desktop-file-utils, wrapGAppsHook, docbook2x, docbook_xml_xslt
+, inform7, gettext, libossp_uuid, gnome2, gnome3, webkit, gtkspell3, libxml2
+, goocanvas2, libplist, gobjectIntrospection, glib, gst_all_1 }:
+
+# Neither gnome-inform7 nor its dependencies ratify and chimara have tagged releases in the GTK3 branch yet.
+
+let
+  ratify = (stdenv.mkDerivation {
+    name = "ratify";
+    src = fetchFromGitHub {
+      owner = "ptomato";
+      repo = "ratify";
+      rev = "f4d2d60ec73d5588e953650b3879e69a727f30ca";
+      sha256 = "eRh/9pYvdfbdbdJQ7pYMLq5p91I+rtyb/AqEGfakjKs=";
+    };
+    nativeBuildInputs = [ meson ninja pkg-config docbook2x docbook_xml_xslt ];
+    buildInputs = [
+      gnome3.gtk
+      gobjectIntrospection
+      gnome3.vala
+      gnome2.gtk-doc
+      wrapGAppsHook
+    ];
+  });
+
+  chimara = (stdenv.mkDerivation {
+    name = "chimara";
+    src = fetchFromGitHub {
+      owner = "chimara";
+      repo = "Chimara";
+      rev = "9934b142af508c75c0f1eed597990f39495b1af4";
+      sha256 = "aRz1XX8XaSLTBIrMIIMS3QNMm6Msi+slrZ6+KYlyRMo=";
+    };
+    nativeBuildInputs =
+      [ meson ninja pkg-config perl bison texinfo python3 glib wrapGAppsHook ];
```

Comment:
```suggestion
    nativeBuildInputs = [ meson ninja pkg-config perl bison texinfo python3 glib wrapGAppsHook ];
```

Reply:
I have reformatted this entire list to one item per line as suggested by jtojnar above.